ilcsoft/test_version.py

```diff
@@ -3,30 +3,30 @@
 ################################################
 
 from .version import Version
-import py
+import pytest
 
 def test_sanity():
-    py.test.raises(TypeError, Version )
-    py.test.raises(ValueError, Version, 0)
-    py.test.raises(ValueError, Version, [])
-    py.test.raises(ValueError, Version, '')
-    py.test.raises(ValueError, Version, '1')
-    py.test.raises(ValueError, Version, '1.')
-    py.test.raises(ValueError, Version, 'a.b')
-    py.test.raises(ValueError, Version, 'blah')
+    pytest.raises(TypeError, Version )
```

Comment:
@andresailer any concerns about this?

Reply:
This works as intended?

Reply:
I think so: https://github.com/iLCSoft/iLCInstall/actions/runs/3638234353/jobs/6140196900#step:5:22 (alternatively, I don't understand pytest reporting)

Reply:
I see "Legacy form" (https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.raises)

Reply:
Ah, right. Just went the easy route here with a `s/py.test/pytest/g` essentially. I can also switch to the the context manager form.

Reply:
Yes, should be OK